pkg/http/api/v1/handler.go

```diff
@@ -226,7 +216,16 @@ func (h *requestHandler) GetMetadata(res http.ResponseWriter, req *http.Request)
 		"com.github.aquasecurity.trivy.severity":      h.config.Trivy.Severity,
 	}
 
-	if !h.config.Trivy.SkipUpdate {
+	vi, err := h.wrapper.GetVersion()
```

Comment:
info: I feel this change is not `chore` since this affects the behavior of API.

Reply:
That's a good point. Shall we bump adapter up to 0.8.0 then?

Reply:
It's up to you.

Reply:
Okey, pushed change to prepare for 0.8.0 release